### PR TITLE
About Page UI Updates

### PR DESCRIPTION
**Pull request by guoz24**
## Description

- UI updates to all tabs of About Page
- Jest tests for all tabs of About Page
- Set up Github Actions to automatically run jest tests on push & PRs into every branch

## Related PRs

https://github.com/RGLab/UITesting/pull/57

## Notion Ticket

https://www.notion.so/rglab/PR-About-Page-UI-Update-871df384652c44e186d28203b6d9644f

## PR Checklist for reviewer

- [ ] Build it on your own local instance and test new features or changes.  
- [ ] Pass UITesting on your local instance on the matching UITesting branch.  
- [ ] It is functional and meets the requirements?   
- [ ] Are any shorthand variable names are included in the [Glossary](https://www.notion.so/rglab/Glossary-of-variable-abbreviations-e205838b1f534abc903fa8c2228a6d7f)?  
- [ ] Does it include descriptive code comments?   
- [ ] Does it follow the [Style Guide](https://github.com/RGLab/LabKeyModules/#style-guide) for any languages included?  
- [ ] Is it well-documented?
    * Module-specific README lists out any dependencies  
    * Sufficient instructions for setup  
    * Links to any relevant LabKey documentation   
- [ ] Are any new features tested via UITesting or jest?  
- [ ] Does it have any new dependencies? Is it justifiable?  
- [ ] Is there code duplication that could be reduced?  
- [ ] Are there performance improvements that could be made?

### CODE REVIEW

File: AboutPage/package.json [hunk 1]
```diff
@@ -18,12 +18,16 @@
...
     "cross-env": "5.2.1",
+    "jest": "27.2.5",
     "prettier": "2.3.1",
+    "pretty": "2.0.0",
     "rimraf": "3.0.2",
+    "ts-jest": "27.0.5",
     "webpack": "5.22.0",
-    "webpack-cli": "4.5.0",
-    "webpack-dev-server": "3.11.2"
+    "webpack-cli": "4.9.0",
```
Comment (helenmiller16) [inline AboutPage/package.json:29]: Why did you update webpack? If you update in this module, it should be updated in all modules to be consistent.

File: AboutPage/src/client/AboutPage/DataReleases.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 interface props {
-  dataReleasesResults: any;
+  dataReleasesResults: JSX.Element | JSX.Element[];
 }
 
 export const DataReleases = React.memo<props>(
   ({ dataReleasesResults }: props) => {
-    const baseUrl = LABKEY.ActionURL["getBaseURL"]();
+    //const baseUrl = LABKEY.ActionURL["getBaseURL"]();
```
Comment (helenmiller16) [inline AboutPage/src/client/AboutPage/DataReleases.tsx:9]: Delete these commented-out lines

**Review by helenmiller16 — COMMENTED** [review]
Looks great, thanks for setting up jest testing! I left a couple of minor inline comments. The only other thing is to delete the extra images in AboutPage/images. Group.png looks like a duplicate of flow.png, and we are no longer using dataflow.jpg. Let's save dataflow.jpg to sharepoint somewhere though so we can easily find it if we ever need to.

File: AboutPage/src/client/AboutPage/DataReleases.tsx (hunk 2 above)
Comment (guoz24) [reply AboutPage/src/client/AboutPage/DataReleases.tsx:9]: Done!

File: AboutPage/package.json (hunk 1 above)
Comment (guoz24) [reply AboutPage/package.json:29]: Discussed in code review. Upgraded to match Analyte Explorer & fix security vulnerabilities.

**Review by helenmiller16 — APPROVED** [review]
Looks good! Just make sure you update software updates page before pushing to Prod.

**Pull request merged**